orttraining/orttraining/test/python/orttraining_test_ortmodule_experimental_json_config.py

```diff
@@ -0,0 +1,105 @@
+
+import os
+import torch
+from onnxruntime.training.ortmodule import ORTModule
+from onnxruntime.capi import _pybind_state as C
+from onnxruntime.training.ortmodule.experimental.json_config import load_from_json
+
+
+class Net(torch.nn.Module):
+    def __init__(self, input_size=784, hidden_size=500, num_classes=10):
+        super(Net, self).__init__()
+
+        self.fc1 = torch.nn.Linear(input_size, hidden_size)
+        self.relu = torch.nn.ReLU()
+        self.fc2 = torch.nn.Linear(hidden_size, num_classes)
+
+    def forward(self, input1):
+        out = self.fc1(input1)
+        out = self.relu(out)
+        out = self.fc2(out)
+        return out
+
+def test_load_config_from_json_1():
+    path_to_json = os.path.join(os.getcwd(), 'orttraining_test_ortmodule_experimental_json_config_1.json')
+
+    device = 'cuda'
+    model = ORTModule(Net().to(device))
+
+    load_from_json(model, path_to_json)
```

Comment:
My previous suggestion was simpler than that. It was a single test in which `load_from_json` is called twice in a row  for each json and assert for the second file. this way, ORTModule default values are replaced by json1 which then changes to json2. If everything asserts to json2, it means all settings from json1 were updated. That only works if json1 and json2 values are all different. That is true, except for the `SaveONNX` which has True both in files

Reply:
Oh I see. If you would like I can change it to that.

Reply:
Made the changes to load files consecutively.